include/LieGroupControllers/impl/ControllerBase.h

```diff
@@ -32,16 +38,46 @@ template <class _Derived> class ControllerBase
     }
 
 public:
+    /**
+     * Set the control state.
+     * @param state of the system.
+     * @return true in case of success, false otherwise.
+     */
     bool setState(const State& state);
 
+    /**
+     * Set the desired state.
+     * @param state of the system.
+     * @return true in case of success, false otherwise.
+     */
     bool setDesiredState(const State& state);
 
+    /**
+     * Set the feedforward term of the controller.
+     * @param feedforward is a vector of the tangent space of the the group.
+     * @return true in case of success, false otherwise.
+     */
     bool setFeedForward(const Vector& feedForward);
 
+    /**
+     * Set the controller gains.
+     * @param gains contains the controller gains.
+     */
     void setGains(const Gains& gains);
 
+    /**
+     * Evaluate the control law.
+     */
     void computeControlLaw();
 
+    /**
+     * Get the control signal.
+     * @return a vector containing the control effort.
+     * @note Please call this function only after computeControlLaw().
+     * @note In the current implementation the control effort belongs to the langet space at the
```

Comment:
tangent space

Reply:
Ah copy paste! I love it!

Reply:
https://github.com/dic-iit/lie-group-controllers/pull/1/commits/311f73d00b482dac6107f9156c19e549733e1ff4